src/codemodder/codemods/test/integration_utils.py

```diff
@@ -62,6 +46,43 @@ class BaseIntegrationTest(DependencyTestMixin, CleanRepoMixin):
     @classmethod
     def setup_class(cls):
         cls.codemod_registry = registry.load_registered_codemods()
+        cls.output_path = tempfile.mkstemp()[1]
+        cls.code_dir = tempfile.mkdtemp()
+
+        if not hasattr(cls, "code_filename"):
+            # Only a few codemods require the analyzed file to have a specific filename
+            # All others can just be `code.py`
+            cls.code_filename = "code.py"
+
+        cls.code_path = os.path.join(cls.code_dir, cls.code_filename)
+
+        if cls.code_filename == "settings.py" and "Django" in str(cls):
```

Comment:
It feels like maybe this should just happen in a Django-specific subclass.

Reply:
very very true. Honestly I was just trying to make as few changes to specific integration test files as possible. Just for now I'm going to leave it as is.